### PR TITLE
Add information about incrementing the build number

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: docs/getting-started/ios/appstore-deployment.md [hunk 1]
```diff
@@ -76,4 +76,23 @@ If you don't have any push certificates already, _get_push_certificate_ will cre
...
 
+<details>
+<summary>Incrementing the build number</summary>
+
+The code sample below will use the latest build number from iTunes Connect and temporarily set it. 
+
+```ruby
+lane :beta do
+  increment_build_number(
+    build_number: app_store_build_number + 1,
```
Comment (janpio) [inline docs/getting-started/ios/appstore-deployment.md]: Where does `app_store_build_number` come from?

Comment (KrauseFx) [reply docs/getting-started/ios/appstore-deployment.md]: It's here https://docs.fastlane.tools/actions/app_store_build_number/

or what do you mean? The testflight action is just an alias

Comment (janpio) [reply docs/getting-started/ios/appstore-deployment.md]: Ah, this is a method that is being called, not a variable that contains some value. 
Confusing or did I miss anything?

Comment (KrauseFx) [reply docs/getting-started/ios/appstore-deployment.md]: Yep, it's an action

Comment (KrauseFx) [reply docs/getting-started/ios/appstore-deployment.md]: What do you propose? multiple lines?

Comment (janpio) [reply docs/getting-started/ios/appstore-deployment.md]: Probably nothing, my PHP brain just wants to see something indicating a function call here: `app_store_build_number()`. But if this is the way it's done in ruby, that's fine - I just didn't get it on first look.

**Pull request merged**